wandb/cli/cli.py

```diff
@@ -2713,7 +2735,14 @@ def online():
     )
 
 
-@cli.command("offline", help="Disable W&B sync")
+@cli.command(
+    "offline",
+    help="""Write metadata generated from W&B runs locally and does not sync to the cloud.
```

Comment:
"Write" and "does not" don't match; should be "write" and "do not".

The words "metadata", "generated", "sync", "cloud" feel unnecessarily technical here. What do you think of "Save data logged to W&B locally without uploading it"? The description should also mention that `wandb sync` is the command to use to upload an offline run.